addons/udes_routing/views/stock_rule_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_stock_rule_form" model="ir.ui.view">
+        <field name="name">Update stock.rule views</field>
```

Comment:
Suggestion: Give the view a code name which mentions the module that is overriding it. E.g. `stock.rule.form.udes_routing` - makes it easier if we ever need to debug views.